clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalCatalog.java

```diff
@@ -95,17 +99,18 @@ public NameIdentifier[] listTables(Namespace namespace) throws NoSuchSchemaExcep
    */
   @Override
   public Table loadTable(NameIdentifier ident) throws NoSuchTableException {
```

Comment:
So, the parameter ident here is the full-qualified name like 'metalake.catalog.schema.table' or `catalog.schema.table` or else?

Reply:
This method is on the catalog level, so the input parameter `ident` just need be a relative path, which is "schema.table". @yuqi1129

Reply:
I see.  I noticed that the Java doc about it stays unchanged, just curious about it.

Reply:
Good suggestion. I can do that in subsequent PRs, together with FilesetCatalog and MessagingCatalog.